spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+describe RentsController, type: :controller do
+  describe 'GET #index' do
+    context 'without authenticated user' do
+      subject!(:http_request) { get :index }
+
+      let!(:rents) { create_list(:rent, 5) }
+
+      it 'responds with status 401 (UNAUTHORIZED)' do
+        expect(response).to have_http_status(:unauthorized)
+      end
+    end
+
+    context 'when fetching all the rents' do
+      include_context 'with authenticated user'
+      subject!(:http_request) { get :index }
+
+      let!(:rents) { create_list(:rent, 5) }
+
+      it 'responds with the books JSON' do
+        expected = ActiveModel::Serializer::CollectionSerializer.new(
+          rents, each_serializer: RentSerializer
+        ).to_json
+        expect(response.body.to_json) =~ JSON.parse(expected)
+      end
+
+      it 'responds with status 200 (OK)' do
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'without any rent' do
+      include_context 'with authenticated user'
+      subject!(:http_request) { get :index }
+
+      it 'responds with an empty body' do
+        expect(JSON.parse(response.body)['page']).to be_empty
+      end
+
+      it 'responds with status 200 (OK)' do
+        expect(response).to have_http_status(:ok)
+      end
+    end
+  end
+
+  describe 'POST #create' do
+    context 'without authenticated user' do
+      it 'responds with status 401 (UNAUTHORIZED)' do
+        post :create, params: FactoryBot.attributes_for(:rent)
+        expect(response).to have_http_status(:unauthorized)
+      end
+    end
+
+    context 'with omitted parameters' do
+      include_context 'with authenticated user'
+      it 'responds to no parameters with status 400 (BAD REQUEST)' do
+        no_parameters = {}
+        expect { post :create, params: no_parameters }.to raise_error(ActionController::ParameterMissing, 'param is missing or the value is empty: rent')
```

Comment:
error_msg = 'param is missing or the value is empty: rent'
expect { post :create, params: {} }.to raise_error(ActionController::ParameterMissing, error_msg)

Reply:
This was one of the lines that were causing me problems with that Rubocop warning.